src/composables/useModalHelper.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * A composable function that provides a method to hide a modal.
+ * @returns {Function} .hide A method that hides the modal.
+ */
+export default function useModalHelper(modalRef) {
+  /**
+   * Hides the modal.
+   */
+  const hide = () => {
```

Comment:
I would change this to hideModal as when its used in a component 'hide' is ambiguous.

Reply:
Done